File: MODUL-6/Bahasa Python/PRAK601-2310817120014-ALIYARAFFANAURAAYU.py
Baris, Kolom = map(int, input().split())
isi = list(map(int, input().split()))
angka = []
for x in range(0, Baris * Kolom, Kolom) :
    angka.append(isi[x : x + Kolom])
print(" ")
for x in range(Baris) :
    for y in range(Kolom) :
        print(angka[x][y], end=' ')
    print()

File: MODUL-6/Bahasa C/PRAK601-2310817120014-ALIYARAFFANAURAAYU.c
#include <stdio.h>

int main (void)
{
    int Baris, Kolom, x, y, isi ;
    scanf("%d %d", &Baris, &Kolom) ;
    int angka[Baris][Kolom] ;
    for (x = 0; x < Baris; x++) {
        for (y = 0; y < Kolom; y++) {
            scanf ("%d", &isi) ;
            angka[x][y] = isi ;
        }
    }
    printf ("\n") ;
    for (x = 0; x < Baris; x++) {
        for (y = 0; y < Kolom; y++) {
            printf ("%d ", angka[x][y]) ;
        }
        printf ("\n") ;
    }
}